nautobot/extras/views.py

```diff
@@ -949,9 +949,12 @@ def _get_job_model_or_404(self, class_path=None, slug=None):
 
     def get(self, request, class_path=None, slug=None):
         job_model = self._get_job_model_or_404(class_path, slug)
-
+        template_name = "extras/job.html"
```

Comment:
While we're updating this view, there's no good reason for `JobView` to not be a subclass of `ObjectDetailView`. Then `template_name` could be set as a class attribute and wouldn't need to be defined in both methods.

@glennmatthews Since you did the work most recently, please keep me honest as to whether this assertion of mine is correct.

Reply:
`JobView` is the view for rendering the Job submission form and submitting that form - it's distinct from the `JobModel` detail view (`JobDetailView`), which *is* an `generic.ObjectView` subclass. We could potentially have `JobView` inherit from `generic.ObjectView` as well but there's a decent amount of behavior of that class that doesn't really apply to this form view IMHO.

Reply:
Yeah thanks for that!